src/api/tojs.ts

```diff
@@ -27,16 +27,17 @@ function toJSHelper(source, options: ToJSOptions, __alreadySeen: Map<any, any>)
     if (!options.recurseEverything && !isObservable(source)) return source
 
     if (typeof source !== "object") return source
+    
+    // Directly return null if source is null
+    if (source === null) return null
```

Comment:
won't it return null already in line#29?

Reply:
@ItamarShDev No, if you run `typeof null` in the console, you will see null is also a unique type of object. I didn't foresee this edge case in my previous PR.

Reply:
thank you very much for the explanation!